policyengine_canada/tests/gov/cra/benefits/dental_benefit/dental_benefit.yaml

```diff
@@ -0,0 +1,58 @@
+- name: One eligible full custody child with afni under 80_000; should be 650
+  period: 2023
+  input:
+    people:
```

Comment:
These tests can be flat since they're at the person level. AFNI can be at the person level; it'll propagate correctly to the household.